users/api/responses.go

```diff
@@ -14,9 +14,6 @@ var (
 	_ mainflux.Response = (*tokenRes)(nil)
 	_ mainflux.Response = (*viewUserRes)(nil)
 	_ mainflux.Response = (*passwChangeRes)(nil)
-	_ mainflux.Response = (*updateGroupRes)(nil)
```

Comment:
What are those changes? I see previous changes that look OK and related to the PR title, but this part is not clear to me.

Reply:
Dead code that I accidentally found :)